java/java.file.launcher/src/org/netbeans/modules/java/file/launcher/actions/SingleJavaSourceRunActionProvider.java

```diff
@@ -81,9 +108,9 @@ public boolean isActionEnabled(String command, Lookup context) throws IllegalArg
         return fileObject != null;
     }
     
-    final LaunchProcess invokeActionHelper (InputOutput io, String command, FileObject fo, ExplicitProcessParameters params) {
+    final LaunchProcess invokeActionHelper (String command, FileObject fo, ExplicitProcessParameters params) {
         JPDAStart start = ActionProvider.COMMAND_DEBUG_SINGLE.equals(command) ?
-                new JPDAStart(io, fo) : null;
+                new JPDAStart(null, fo) : null;
```

Comment:
Can probably change `JPDAStart` to not require the `InputOutput`?  It's hardly used.  But could NPE with this.

Reply:
I don't se a NPE, it works without a visible difference.

Reply:
eg. https://github.com/apache/netbeans/blob/master/java/java.file.launcher/src/org/netbeans/modules/java/file/launcher/actions/JPDAStart.java#L126